packages/next/build/webpack-config.ts

```diff
@@ -1668,8 +1668,13 @@ export default async function getBaseWebpackConfig(
     },
   }
 
+  if (!webpack5Config.output) {
+    webpack5Config.output = {}
+  }
+  webpack5Config.output.trustedTypes = 'nextjs#bundler'
```

Comment:
```suggestion
  if (isClient) {
    webpack5Config.output.trustedTypes = 'nextjs#bundler'
  }
```
Can we only enable this for the client build? I don't think it's needed for the server/edge builds